src/app/pages/AccountPage/Features/AccountSummary/index.tsx

```diff
@@ -79,59 +80,24 @@ export function AccountSummary(props: AccountSummaryProps) {
         )}
       </Box>
       {walletIsOpen && walletAddress === address && (
-        <Box
-          border={{
-            color: 'status-ok',
-            side: 'left',
-            size: '3px',
-          }}
-          background={{
-            color: 'status-ok',
-            opacity: 'weak',
-          }}
-          pad={{ horizontal: 'small', vertical: 'xsmall' }}
-        >
+        <AlertBox color="status-ok">
           <Text weight="bold">{t('account.summary.yourAccount', 'This is your account.')}</Text>
```

Comment:
You can remove `<Text weight="bold">` as it's already handled by AlertBox

Reply:
Good catch!